tools/floodgate/js/floodgate.js

```diff
@@ -28,7 +28,7 @@ async function floodgateContentAction(project, config) {
 
 async function triggerUpdateFragments() {
   loadingON('Fetching and updating fragments..');
-  const status = await updateFragments(initProject);
+  const status = await updateFragments(initProject, true);
```

Comment:
Personally, I'm not a big fan of sending Boolean values as arguments, since one wouldn't know what they refer to just by looking at where the function is called. Replacing it with a variable, adding a comment or switching the list of arguments with a config object might be better alternatives for future readers of this code

Reply:
Updated. Thanks @overmyheadandbody